### ISSUE TITLE
Test for preferred_address uses an invalid format

### ISSUE THREAD

**Issue by jlaine** [kazu-yamamoto/h3spec#1]
The test which assert a server rejects transport parameters with a preferred_address uses a bogus format. This is likely to result in triggering a parsing error instead of just checking the server rejects the presence of this parameter.

I think malformed parameters are also an interesting case, but they should be kept separate.

**Comment by kazu-yamamoto** [comment]
Is the following data good enough?

```
"7f000001"
"01bb"
"00000000000000000000000000000000"
"0000"
"08"
"0001020304050607"
"000102030405060708090a0b0c0d0e0f"
```

**Comment by jlaine** [comment]
Sure that looks fine!

**Comment by jlaine** [comment]
Fixed in 7c5ce12ee7eef4b248dae5c99c319ca668ba5f09
